### DISCUSSION TITLE
Not Receiving My GitHub Education Benefits After Approved

### DISCUSSION THREAD

**Discussion by YuzhenQin1** [community/community#145648] [GitHub Education]
### Hi everyone,

I was approved on Nov. 24, and my benefits are still not available.

Just wondering if the GitHub Student Pack is provisioned manually? Since, seemingly, the GitHub Education applications are approved automatically.

![image](https://github.com/user-attachments/assets/fc338532-99c1-4e4a-b0b1-f8aea267ae29)

**Comment by davevad93** [comment]
Hi @YuzhenQin1 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @YuzhenQin1 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by maheshkeswani** [comment]
Hi @davevad93 , i m also facing same problen , can u please have a look into it , I have submitted it on 25 Nov. 2024 IST .

**Comment by billy122333** [comment]
Hi @davevad93 , i m also facing the same problem , can u please have a look into it , I have submitted it on 25 Nov. 2024 IST .

**Comment by 2018wzh** [comment]
Hi @davevad93 , i m also facing the same problem , can u please have a look into it , I have submitted it on 26 Nov. 2024 IST .

**Comment by maruf1020** [comment]
Hi @davevad93, I'm facing the same issue. Could you please take a look at it? I submitted it on November 26, 2024.

**Comment by davevad93** [comment]
Hey @maheshkeswani , @billy122333 , @2018wzh and @maruf1020 , sorry but I'm not a GitHub employee so I can't take a look at your application, the above answer is a guideline for users since it's a question that is often asked here in the Community. My best advice is to be patient as the process can take some time and GitHub no longer provides manual reviews. 👍
